dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -149,7 +149,7 @@ emptyServletMappings.name=Return empty servlet mappings
 emptyServletMappings.desc=Toggle if you want to return an empty collection, instead of null, when no servlet mappings are added. The default value is false. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.emptyservletmappings.
 
 deferServletRequestListenerDestroyOnError.name=Defer servlet request listener destroy
-deferServletRequestListenerDestroyOnError.desc=Toggle if you want to defer ServletRequestListener destroy when there is an error serving the request. The default value is false. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.deferServletRequestListenerDestroyOnError.
+deferServletRequestListenerDestroyOnError.desc=Toggle if you want to defer ServletRequestListener destroy when there is an error serving the request. The default value is false. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.deferServletRequestListenerDestroyOnError. For servlet-5.0 and newer Servlet features, the default value is true.
```

Comment:
Either remove "The default value is false." or, preferably, change the description to:

Toggle if you want to defer ServletRequestListener destroy when there is an error serving the request. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.deferServletRequestListenerDestroyOnError. For servlet-5.0 and newer Servlet features, the default value is true. For older Servlet features, the default is false.

Reply:
@helyarp deferServletRequestListenerDestroyOnError is an old property (has been out there many years).  I don't know whether any customer may use any of these texts in their automation (they may not use or rely on the  description ...but I just don't know).

If it is necessary, I will make a change; but if you think it is ok, we should just leave it as-is.